src/utils/markdown/rehype-unicorn-iframe-click-to-run.ts

```diff
@@ -139,7 +139,9 @@ export const rehypeUnicornIFrameClickToRun: Plugin<
 							: undefined,
 						"data-width": width,
 						"data-height": height,
-						style: `height: ${height}px; width: ${width}px;`,
+						style: `height: ${+height ? `${height}px` : height}; width: ${
+							+width ? `${width}px` : width
+						};`,
```

Comment:
Let's use `Number()` instead of `+`:

```suggestion
					style: `height: ${Number(height) ? `${height}px` : height}; width: ${
							Number(width) ? `${width}px` : width
						};`,
```